Index.html

```diff
@@ -14,38 +14,36 @@
 <body>
     <header class="toolbar">
         <a class="logo">Welcome</a>
-        <nav class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></nav>
-        
+        <nav class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></nav>        
     </header>
     <section class="headline">
+        
+        <br class="hb-01">
         <h1>
             Hey there. I'm Cindy
         </h1>
         <h2 class="greenheader">
             I'm a software developer
-        </h2>
-        
-        <p>
+        </h2>        
+        <p class="p-headline">
             I can help you build a product, feature or website. Look through some of my work and experience! 
             If you like what you see and have a project you need coded, don't hesitate to contact me.
         </p>
-        
-        <ul>
+        <ul class="social-heading">
             <li><a href="https://twitter.com/CindyDorantes10"><img src="./Images/twitter.png" alt="Twitter icon" class="socialicon"></a></li>
             <li><a href="https://www.linkedin.com/in/cindydorantessanchez/"><img src="./Images/Linkedin_icon.png" alt="LinkedIn icon" class="socialicon"></a></li>
             <li><a href="https://medium.com/"><img src="./Images/medium.png" alt="Medium icon" class="socialicon"></a></li>
             <li><a href="https://github.com/CindyDorantes"><img src="./Images/github.png" alt="GitHub icon" class="socialicon"></a></li>
             <li><a href="https://angel.co/"><img src="./Images/angellist.png" alt="AngelList icon" class="socialicon"></a></li>
```

Comment:
- [ ] After the first section, there is more spacing on the button so kindly increase the spacing there: Here is the figure:
### your design
![your design](https://user-images.githubusercontent.com/81727988/172988783-73088261-fe6e-4f45-a187-b7294559059d.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/172988819-231eb289-d8e3-4c7d-8a86-845f63ad8804.png)